### ISSUE TITLE
Add short gene names (symbols) to region view tables

### ISSUE THREAD

**Issue by amkwong** [statgen/fivex#71]
Currently, the data the powers region view tables comes from credible_sets, which goes through a data ingestion step to add extra columns of data (such as P-values and effect sizes). We want to also show short gene names in this table. The easiest way is to add an extra column in the data file so that querying the credible_sets file will yield all the necessary information. 

We will also need to update the Tabulator table to display this extra column, and enable text box filtering at the top of the gene column.

Minor changes: 
The current label "Gene" should be changed to "Gene ID", and the new column should be named "Gene"
The auto-filled filter should use the short gene name with the new column

tl;dr: In the data ingestion pipeline, add a column for short gene names (e.g. "SORT1") corresponding to the Ensembl gene ID (e.g. "ENSG00000134243")
